recipes/pbzip2/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "1.1.13" %}
+{% set major_version = "1.1" %}
+{% set sha1 = "f61e65a7616a3492815d18689c202d0685fe167d" %}
+{% set md5 = "4cb87da2dba05540afce162f34b3a9a6" %}
+
+package:
+  name: pbzip2
+  version: {{ version }}
+
+source:
+  fn: pbzip2-{{ version }}.tar.gz
+  url: https://launchpad.net/pbzip2/{{ major_version }}/{{ version }}/+download/pbzip2-{{ version }}.tar.gz
+  sha1: {{ sha1 }}
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - bzip2
```

Comment:
This package statically links to bzlib, so it needs bzlib.h at compile time (see circleci tests for 5ec8db1). It rolls what it needs into the executable at build time so that it doesn't need external bzlib files lying around at runtime.